neo/SmartContract/ApplicationEngine.py

```diff
@@ -292,10 +292,19 @@ def GetPrice(self):
             return 20
         elif opcode == CHECKSIG:
             return 100
+        elif opcode == VERIFY:
+            return 100
         elif opcode == CHECKMULTISIG:
             if self.EvaluationStack.Count == 0:
                 return 1
-            n = self.EvaluationStack.Peek().GetBigInteger()
+            
+            item = self.EvaluationStack.Peek()
+
+            array = isinstance(item, list)
```

Comment:
You'll need to check for the interop `Array` type, not the built in `list` type.
```
if isintance(item,Array):
   n = item.Count
```

Reply:
I can't believe it is as easy as just checking if it is an Array lol.